### PR TITLE
data:Ineva-A

### PR DESCRIPTION
**Pull request by Ineva-A**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3553"></a>

### CODE REVIEW

File: data'/Ineva-A.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "WebDev, DevRel. Motivated to Programm, self learner",
+  "avatar": "https://github.com/Ineva-A.png",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/ineva_yt",
+      "icon": "FaTwitter"
+    }
```
Comment (arpitghura) [inline data'/Ineva-A.json:12]: Add `,` after each object
```suggestion
    },
```

File: data'/Ineva-A.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/regina-ganser-551b26244/",
+      "icon": "FaLinkedIn"
```
Comment (eddiejaoude) [inline data'/Ineva-A.json:16]: I think the `i` is lowercase, `FaLinkedin`

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for contributing 👍 the folder name should not have ` '` in it

note: `data` folder already exists

**Review by arpitghura — CHANGES_REQUESTED** [review]

**Comment by Ineva-A** [comment]
Ah, i see. will change it

**Pull request closed without merging**